### PR TITLE
Update take_screenshot method

### LINKED ISSUE #3115: Update take screenshot procedure according new naming conventions
It seems that we will not specify module name for test cases name, so in result we will have a lot of `test_positive_create` or `test_positive_update` methods that will make our screenshot procedure very hard to process and analyze. It is necessary to improve it

### PR DESCRIPTION
**Pull request by elyezer**
In order to better identify the screen shots now the path have the class
name as a component. The new name will be
screenshots_path/YYYY-MM-DD/ClassName/test_method_name/filename.png.

Closes #3115

### CODE REVIEW

**Comment by sghai** [comment]
ACK.. Looks good.

File: robottelo/test.py [hunk 1]
```diff
@@ -221,22 +220,23 @@ def setUp(self):  # noqa
         self.user = User(self.browser)
         self.usergroup = UserGroup(self.browser)
 
-    def take_screenshot(self, testmethodname):
+    def take_screenshot(self):
         """Takes screenshot of the UI and saves it to the disk by creating
```
Comment (oshtaier) [inline robottelo/test.py]: Please, update docstring accordingly

**Comment by oshtaier** [comment]
ACK pending comment

File: robottelo/test.py (hunk 1 above)
Comment (elyezer) [inline robottelo/test.py]: :+1:

**Comment by elyezer** [comment]
Comments addressed and rebased.

**Comment by oshtaier** [comment]
ACK

**Comment by oshtaier** [comment]
Oops. I had second ACK. Sorry @elyezer

**Pull request merged**